e2e/autoscript-suites/src/utilities/setup-and-go.ts

```diff
@@ -80,7 +80,9 @@ export async function setupAndGo(
     await dialog.accept(config?.dialogInput || 'abc123');
   });
 
-  await page.waitForSelector(config?.selector || '.Test_Complete');
+  await page.waitForSelector(config?.selector || '.Test_Complete', {
+    state: 'visible',
+  });
```

Comment:
I believe `state: visible` is the default, so this is optional. Is this a leftover from your debugging?